pages/bucketList.tsx

```diff
@@ -1,18 +1,37 @@
 import styles from '@/styles/pageStyles/bucketList.module.css'
 import dynamic from 'next/dynamic'
+import { testLocations } from '@/constants/test'
+import { useEffect, useState } from 'react'
+import BucketListDisplay from '@/components/BucketListPage/BucketListDisplay'
 
 
 function bucketList(props: any) {
   const NavBar = dynamic(() => import('@/components/NavBar'))
   const BucketListDefault = dynamic(() => import('@/components/BucketListPage/BucketListDefault'))
 
+  const [locationData, setLocationData] = useState({})
+  const [display, setDisplay] = useState(false)
+  const testData = testLocations
+
+  useEffect(() => {
+   if(testData){
```

Comment:
Not a HUGE fan of this method of testing just cause it'll require us to remember to remove it 

Although we HAVE to for it to work, so I guess not a big deal